### PR TITLE
Fix grafana boards

### PR DESCRIPTION
**Pull request by remk**
Quotes query to avoid all the panel of a board using queries containing the same prefix followed by an `-` to display the same values.

Plus fix the IMAP CREATE metric reporting

### CODE REVIEW

File: grafana-reporting/MATCHER-1490071813409-dashboard.json [hunk 1]
```diff
@@ -117,7 +117,7 @@
                   "type": "avg"
                 }
               ],
-              "query": "name:All",
+              "query": "name:\"All\"",
```
Comment (rouazana) [inline grafana-reporting/MATCHER-1490071813409-dashboard.json]: do you know if it's a bad named data or if it's a specific keyword?

Comment (remk) [reply grafana-reporting/MATCHER-1490071813409-dashboard.json]: It is for the `All` matcher.

**Comment by Arsnael** [comment]
test this please

**Comment by chibenwa** [comment]
I think this can be merged without waiting a :green_apple:  build.

**Pull request merged**